parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -255,17 +255,17 @@ configureK8s() {
         set -x
     fi
 
+    set +x
     if [ "${ENABLE_SECURE_TLS_BOOTSTRAPPING}" = "false" ] && [ -z "${TLS_BOOTSTRAP_TOKEN:-}" ]; then
         # only create the client cert and key if we're not using vanilla/secure TLS bootstrapping
-        set +x
```

Comment:
We need the +x and -x to stop the cert content getting logged.

Reply:
Yes I know. I extended the block where the loggin is disabled.

`IF` statement logged the token and broke some tests. Not sure why it didn't fail before